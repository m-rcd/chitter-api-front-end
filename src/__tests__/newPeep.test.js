import React from 'react';
import renderer from 'react-test-renderer';
import NewPeep from '../components/new_peep';

it('NewPeep Snapshot', () => {
  const tree = renderer.create(<NewPeep />).toJSON();
  expect(tree).toMatchSnapshot();
});
