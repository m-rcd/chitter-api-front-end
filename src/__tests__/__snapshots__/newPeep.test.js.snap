// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewPeep Snapshot 1`] = `
<form>
  <input
    name="peep"
    type="text"
  />
  <input
    type="submit"
    value="Submit"
  />
</form>
`;
